tailwind.config.js

```diff
@@ -10,6 +11,15 @@ module.exports = {
       colors: {
         bg: {
           dark: "hsl(240, 8%, 3%)",
+          darkShaded: "hsl(220, 19%, 8%)",
+          lightBluePrimary: "hsl(220, 100%, 58.4%)",
+          lightSecondary: "hsl(0, 0%, 100%)",
+          lightGray: "hsl(215, 33.3%, 92.9%)"
```

Comment:
Sidenote that we should work with the designers to set up some colours.

Tailwind works much better when you have one colour palette, not a separate one for borders and backgrounds.

Right now we have this weird syntax where are classes repeat themselves a bit, e.g. `bg-bg-dark` vs just `bg-grey-900` and `text-font-darkPrimary` instead of `text-primary-dark`, etc.

Reply:
Agreed, I do think it's weird and would like to systemize this more. Btw, `bg-bg-dark` refers to a tone of color, not dark  as in bold (`900`)